modin/pandas/dataframe.py

```diff
@@ -1637,7 +1630,6 @@ def prod(
         Return the product of the values over the requested axis.
         """
         axis = self._get_axis_number(axis)
-        validate_bool_kwarg(skipna, "skipna", none_allowed=False)
```

Comment:
Shouldn't we use `self.validate_bool_kwarg` here?

Reply:
good point, fixed